servers/parser-graphql-wrapper/src/preview/preview.ts

```diff
@@ -0,0 +1,93 @@
+import { Item, ItemSummary } from '../model';
```

Comment:
I'm dreaming of a day where we have an OpenGraph entity, a Parser Item Entity, and a Model which transforms the fields to be returned by the resolvers...

Reply:
sammme